packages/dd-trace/src/config.js

```diff
@@ -244,13 +247,31 @@ class Config {
 
     // Configure the logger first so it can be used to warn about other configs
     const logConfig = log.getConfig()
-    this.debug = logConfig.enabled
+    this.debug = isTrue(coalesce(
+      fleetFileConfigEntries.DD_TRACE_DEBUG,
+      process.env.DD_TRACE_DEBUG,
+      process.env.OTEL_LOG_LEVEL === 'debug' || undefined,
+      localFileConfigEntries.DD_TRACE_DEBUG,
+      logConfig.enabled
+    ))
     this.logger = coalesce(options.logger, logConfig.logger)
-    this.logLevel = coalesce(options.logLevel, logConfig.logLevel)
+    this.logLevel = coalesce(
+      options.logLevel,
+      fleetFileConfigEntries.DD_TRACE_LOG_LEVEL,
+      process.env.DD_TRACE_LOG_LEVEL,
```

Comment:
This is now defined in 2 places as the logger reads it directly.

Reply:
Similar concern for the above as well.

Reply:
Done in [address review](https://github.com/DataDog/dd-trace-js/pull/5126/commits/4e83b16fa4e1e31c7f3ef42c9e4f71f13287c12f)